docs/tutorials/getting_started/tutorial_create_expectations.md

```diff
@@ -101,9 +101,9 @@ Since notebooks are often less permanent, creating Expectations in a notebook al
 ```
 
 **Cell 3**
-![Cell3](../../images/getting_started_tutorial/tutorial_create_expectations_cell3_onboarding_assistant.png)
+![Cell3](../../images/getting_started_tutorial/tutorial_create_expectations_cell3.png)
```

Comment:
@Rachel-Reverie I just noticed a few items to change in this image, see my friction log for more details.